### ISSUE TITLE
[Bug]: After import, clicking on commit causes the 360 app to crash.

### ISSUE THREAD

**Issue by btsgh** [appsmithorg/appsmith#33893]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

Once the app is imported, on opening the app and clicking on the commit icon causes the app to crash

### Steps To Reproduce

1. Git import the 360 app 
2. Once you refresh and see it in the home page, click on commit icon
3. Note that it waits on this progress bar for about a minute and then the app crashes

### Environment

Deploy Preview

### Severity

High (Blocker to building or releasing)

### Issue video log

https://jam.dev/c/3c6944f0-21fe-43f3-8b31-df49211bd0c7

### Version

Cloud DP

**Comment by btsgh** [comment]
This issue I still see the very first time I import the 360 app. Attaching video for reference. 
https://jam.dev/c/c9afeac2-8b1e-49af-9d39-6ef87184d979
@sondermanish

**Comment by btsgh** [comment]
Works fine. I am able to commit the changes in the 360 app [1.9 mb on disk] without the app crashing. Also tried this with Templates app which is of a similar size.

**Comment by btsgh** [comment]
Works fine. I am able to commit the changes in the 360 app [1.9 mb on disk] without the app crashing. 

Tried this with Templates app which is of a similar size. Saw some First Commits, and when I clicked on commit, here I see the commit modal sitting on the same screen, and does not go through. 
On refresh, the commits disappear, and there are no commits on the repo either! When I add some commits and try to commit again, it comes up again as First Commit and goes through fine.
